glyph/src/ttfp.rs

```diff
@@ -341,9 +359,13 @@ macro_rules! impl_font {
                     },
                 })
             }
+
+            fn font_data(&self) -> &[u8] {
+                self.$font_data()
+            }
         }
     };
 }
 
-impl_font!(FontRef<'_>);
-impl_font!(FontVec);
+impl_font!(FontRef<'_>, font_data);
+impl_font!(FontVec, as_slice);
```

Comment:
I wonder if `as_slice` is exactly the same as `font_data` always? Ie when using data + index.

So I think we should use upstream `raw_face().data` for Vec & Ref to ensure consistency.

Reply:
That makes sense, done.